copy/copy.go

```diff
@@ -361,6 +365,44 @@ func supportsMultipleImages(dest types.ImageDestination) bool {
 	return false
 }
 
+// compareImageDestinationManifestEqual compares the `src` and `dest` image's manifests, returning the 'true' if they compare equal.
```

Comment:
`checkIfDestinationManifestIsEqual` ?

“returning true and information about the manifest if…”